config/janitor.php

```diff
@@ -37,6 +37,7 @@
 
         'passport' => [
             'oauth_token_url' => env('JANITOR_OAUTH_TOKEN_URL', 'http://localhost/oauth/token'),
+            'client_model' => \Laravel\Passport\Passport::class
```

Comment:
This should be `\Laravel\Passport\Client::class`, also make sure to add the trailing comma required by StyleCI.

Reply:
Thanks. Pushed up those changes and passes tests.